index.html

```diff
@@ -1,61 +1,117 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/d7c4625610.js" crossorigin="anonymous"></script>
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+    <title>Common Web | Homepage</title>
 </head>
+
 <body>
+    <header>
+        <nav class="nav01">
+          <ul class="desktop-bar">
+                <li><i class="fab fa-facebook-f"></i></li>
+                <li><i class="fab fa-twitter"></i></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </nav>
+
+        <i class="fas fa-bars fa-2x"></i>
+        <nav class="nav02">
+            <a href="index.html" class="top-logo" ><img src="assets/images/web-logo.png" alt="logo" class="top-logo"/></a>
+            <ul class="nav-links">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html" target="_blank">About</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">Courses</a></li>
+            </ul>
+        </nav>
+    </header>
     <section class="headline">
+
+        <p class="greetings">
+            "Hello! Welcome our humble abode"
+        </p>
         <h1>
-            WEB DEVELOPMENT FOR THE COMMON MAN
+            WEB FOR THE COMMON MAN - 2022
         </h1>
 
-        <p>
+        <p class="intro">
+            <em><strong>'Web of the common man, by the common man and for the common man'
+                </strong></em>
+            <br>
             A chance for all and sundry to own the knowlegde of the web
             no matter what your background might be.
         </p>
 
-        <p> <strong>24/7 UNLIMITED ACCESS FOR FREE!</strong>
-            
+        <p class="introt"> <strong>24/7 UNLIMITED ACCESS FOR FREE!</strong>
+
         </p>
-        <p>Get your mobile devices ready and let rock 'n' roll!!!</p>
-        
+        <p class="introt">Get your mobile devices ready and let's rock 'n' roll!!!</p>
+
     </section>
 
     <section class="program-list">
         <h2>Main Program</h2>
-        
-        <ul class="items">
+        <hr>
+        <ul class="program-items">
+            <li>
+                <img src="assets/images/html.png" alt="#" class="html">
+                <h4>HTML5</h4>
+                <p>a markup language used for structuring and presenting content on the World Wide Web.</p>
+            </li>
+            <li><img src="assets/images/css.png" alt="#" class="html">
+                <h4>CSS3</h4>
+                <p>Used in the styling and formatting of Web pages. </p>
```

Comment:
- [ ] Also, related to your images, realize that your CSS image container and content container are smaller than the rest of the containers for this `ul`, which again, does not follow the given template and can lead to a bad user experience as well. Kindly take a look at it.
_Note: This is for both desktop and mobile version_

Reply:
fixed it